.verb.md

```diff
@@ -45,28 +33,9 @@ strip(commentString, { preserveNewlines: true });
 {%= include("author") %}
 
 ## License
-{%= copyright() %}
+{%= copyright({linkify: true, start: 2014}) %}
 {%= license %}
 
 ***
 
 {%= include("footer") %}
-
-
-```js
-//example.strip
-console.log(strip("foo // this is a comment\n/* me too */"));
-//=> 'foo'
-```
```

Comment:
i know, but i think it make more sense to be in the code comment and to be absolutely simple and minimal. that's the idea of code comments - help. :)